### ISSUE TITLE
Inconsistent behavior when reverse generating views from a DB Project versus a Database.

### ISSUE THREAD

**Issue by DDtMM** [ErikEJ/EFCorePowerTools#747]
When reverse generating from a db project, certain columns are not getting generated in the model class.  If I reverse the database from a database instance that originated from that project then the columns do appear in the model.

### Steps to reproduce
1. Create a view where some columns directly reference columns from other tables and some columns are the results of functions such as a query with an aggregate:

```
CREATE VIEW [dbo].[ContractHistory]
	AS
SELECT ContractId, MAX(SysStart) AS SysStartMax
FROM Contract
GROUP BY ContractId
```

2. Run _EF Core Power Tools_ > _Reverse Engineer_.  
3. Select your database project as Database Connection.  
4. Make sure the View is selected in the next screen (notice that SysStartMax isn't present when expanding on view). 
5. HIt OK.

The following model is generated.

```
public partial class ContractHistory
{
    public int ContractId { get; set; }
}
```

6. Deploy the database project to a server.
7. Repeat steps 2-5, but connect to the deployed database in step 3 (In step 4 SysStartMax is present).

The following model is generated:

```
public partial class ContractHistory
{
    public int ContractId { get; set; }
    public DateTime? SysStartMax { get; set; }
}
```

### Further technical details
EF Core Power Tools version: 2.5.454.0
Database engine: SQL Server
Visual Studio version: 2019 16.8.3

I did a lot of experiments to see what would and what wouldn't get generated.  When I looked at the _model.xml_ file from the _.dacpac_ archive generated from the built project it seemed that there was a pattern.  All of the columns would get represented under a **Relationship** node with a _Name_ attribute set to _"Columns"_, but only those **Element** nodes with a **Relationship** inside of them would get generated.

```
<Relationship Name="Columns">
	<Entry>
		<!-- This will get generated -->
		<Element Type="SqlComputedColumn" Name="[dbo].[ContractHistory].[ContractId]">
			<Relationship Name="ExpressionDependencies">
				<Entry>
					<References Name="[history].[Contract].[ContractId]" />
				</Entry>
			</Relationship>
		</Element>
	</Entry>
	<Entry>
		<!-- This WILL NOT get generated -->
		<Element Type="SqlComputedColumn" Name="[dbo].[ContractHistory].[StartsOn]" />
	</Entry>
</Relationship>
```

**Comment by ErikEJ** [comment]
I have no way of getting the column name, and without it, EF Core has no way of selecting the column from the view. 
I think the conclusion is that view support for .dacpac has limitations. Maybe I should make it clearer in the docs? (I think it is not really mentioned at the moment)

```xml
<Entry>
		<!-- This WILL NOT get generated -->
		<Element Type="SqlComputedColumn" Name="[dbo].[ContractHistory].[StartsOn]" />
	</Entry>
```

**Comment by ErikEJ** [comment]
Ping?

**Comment by DDtMM** [comment]
Once I mentally tried going through the exercise of determining other necessary metadata for the columns from the dacpac file I realized the challenge.  I did see this `SO Question`[https://stackoverflow.com/questions/40747082/finding-data-type-information-for-certain-objects-in-dacfx-api], but I lack the understanding to determine if it is helpful or even relevant.

Amending the documentation would be sufficient.  Perhaps suggesting adding missing columns would a partial class would be a nice note to add?  Also, a warning in the UI would be helpful when selecting views from with a SQL Server project.

Thank you very much for addressing this issue.
